nginx/assets/monitors/4xx.json

```diff
@@ -0,0 +1,32 @@
+{
+	"name": "[NGINX] 4xx Errors",
+	"type": "query alert",
+	"query": "avg(last_4h):anomalies(avg:NGINX.upstream.peers.responses.4xx{*}.as_count(), 'basic', 2, direction='above', alert_window='last_15m', interval=60, count_default_zero='true') >= 1",
+	"message": "Number of 4xx errors on NGINX upstreams is at {{value}} which is higher than usual.",
+	"tags": [
+		"integration:nginx"
+	],
+	"options": {
+		"notify_audit": false,
+		"locked": false,
+		"timeout_h": 0,
+		"new_host_delay": 300,
+		"require_full_window": true,
```

Comment:
This should be false (instructions were updates).

Reply:
fixed